source/beamline/AMSlit.h

```diff
@@ -0,0 +1,144 @@
+#ifndef AMSLIT_H
+#define AMSLIT_H
+
+#include "beamline/AMControl.h"
+#include "beamline/AMSlitControl.h"
+#include "beamline/AMControlSet.h"
+
+#define AMSLIT_CANNOT_OPEN 39810
+#define AMSLIT_CANNOT_CLOSE 39811
+
+class AMSlit : public AMControl
```

Comment:
is it possible for this to subclass from `AMSlitControl`, since it is a pseudo control and also has `firstBlader_` and `secondBlader_`?